### ISSUE TITLE
[FEAT]: Expert Train & Inference for NY Cartoon dataset with ALBEF

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#96]
### Description

Use NY Cartoon training split of R, U, and S to train model ALBEF seperately and save to

Sarc R finetuned ALBEF
Sarc U finetuned ALBEF
Sarc S finetuned ALBEF
Then record the performance of the three models on the test set.
